src/test/ui/nll/closure-requirements/propagate-fail-to-approximate-longer-wrong-bounds.stderr

```diff
@@ -8,12 +8,12 @@ error: unsatisfied lifetime constraints
   --> $DIR/propagate-fail-to-approximate-longer-wrong-bounds.rs:51:24
    |
 LL |     establish_relationships(&cell_a, &cell_b, |_outlives1, _outlives2, x, y| {
-   |                                                ----------  ---------- has type `&'4 std::cell::Cell<&'5 &'6 u32>`
+   |                                                ----------  ---------- has type `&'_ std::cell::Cell<&'2 &'_ u32>`
    |                                                |
-   |                                                has type `&'1 std::cell::Cell<&'2 &'3 u32>`
+   |                                                has type `&'_ std::cell::Cell<&'1 &'_ u32>`
```

Comment:
as an aside, it might be nice to not print `'_` in this case but just `&`

Reply:
Updated most recent commit to address this.